velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -393,13 +403,30 @@ velox::test::ResultOrError AggregationFuzzerBase::execute(
       builder.splits(splits);
     }
 
+    ScopedOOMInjector oomInjector(
+        []() -> bool { return folly::Random::oneIn(10); },
+        10); // Check the condition every 10 ms.
+    if (FLAGS_enable_oom_chaos_testing) {
+      oomInjector.enable();
+    }
+
     TestScopedSpillInjection scopedSpillInjection(spillPct);
     resultOrError.result =
         builder.maxDrivers(maxDrivers).copyResults(pool_.get());
   } catch (VeloxUserError& e) {
     // NOTE: velox user exception is accepted as it is caused by the invalid
     // fuzzer test inputs.
     resultOrError.exceptionPtr = std::current_exception();
+  } catch (VeloxRuntimeError& e) {
+    if (FLAGS_enable_oom_chaos_testing &&
+        e.errorCode() == facebook::velox::error_code::kMemCapExceeded) {
```

Comment:
Shall we also check the error message as well so it won't be caused by the real query oom? Thanks!